src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  if (days === 7) {
+    return 230;
```

Comment:
this is hardcode. You should calculate value you need to return depends on an argument you get and discounts